docs/PersonalApplications.rst

```diff
@@ -314,6 +318,64 @@ The activities in My Activities are similar to those in :ref:`My Activities <Man
 			like on your activity stream page. However, only your connections
 			can see the Activity Composer that enables them to post on your stream.
 
+.. _Managing-Fav-Apps:
+
+=======================================
+Managing your favourite applications
+=======================================
+
+Starting with eXo Platform 6, you can quickly access your favourite applications 
+through the Applications drawer.
+
+To access your favourite applications, you just need to click on this button |AC_image0| from
+the top navigation bar.
+
+|AC_image1|
+
+A drawer will open displaying your favourite applications alphabetically sorted.
+
+|AC_image2|
+ 
+You can reorganize your favorite applications by dragging and dropping their icons according to your choice.
+
+|AC_image3|
+ 
+.. note:: When an administrator sets some applications as default, they also appear in your applications
+          center drawer.
+
+When you click on an application icon it will open in the same tab of it is an eXo Platform, otherwise,
+it will open in a new tab.
+
+In addition to your favorite application displayed in the drawer, you can also navigate through the whole 
+applications list you are allowed to view.
+
+You just need to click on |AC_image4| button in the drawer to access the applications center.
+
+|AC_image5|
+
+Application center is devided into 2 parts: on at the right listing your favorite applications and 
+the second listing the whole applications your are allowed to view and use.
+
+This second part contains applications cards with these details for each one:
+
+-  Application's name and image
+-  Application's description
+-  Two buttons "Open" and |AC_image6| in the card's bottom
+-  The icon |AC_image7| in the card's topper
+
+You can favorite an application in the application center by clicking on the |AC_image8| in the application's card.
+The "Star" button will then be yellow colored and the application will be automatically added to the right panel
+i.e. to the "favorite applications" list and to the drawer.
+
+You can also unfavorite an application in the applications center either by clicking on the yellow colored 
```

Comment:
```suggestion
You can also unfavorite an application in the applications center either by clicking on the yellow colored 
```
"applications center" or "Application Center" ?

Reply:
Application Center, I'll add another commit to align nominations